browser-extension/src/popup/components/content/list.config.tsx

```diff
@@ -30,11 +32,20 @@ export const LIST_HEADERS: ListHeader[] = [
   },
 ];
 
+const onEditClick = async (item) => {
+  const url: string = chrome.runtime.getURL(`options/options.html#/edit/${item.pageType}/${item.id}`);
+  await TabService.focusOrCreateTab(url);
```

Comment:
lets use ```TabService.createTab``` function instead

Reply:
Why don't active tab at first, if tab already has?

Reply:
since using an already created tab has no advantage over creating a new tab. And for getting already created new tab needs additional functionality